src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetStandardDetailsByStandardUIdRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests
+{
+    public class GetStandardDetailsByStandardUIdRequest : IGetApiRequest
```

Comment:
Can we merge this with the requests that operate by LarsCode?

Reply:
you do it 😉